src/services/kuma-api/FakeKuma.ts

```diff
@@ -0,0 +1,46 @@
+import { Faker } from '@faker-js/faker'
+
+export class KumaModule {
+  faker: Faker
+  constructor(
+    faker: Faker,
+  ) {
+    this.faker = faker
+  }
+
+  serviceType() {
+    return this.faker.helpers.arrayElement(
+      [
+        'internal',
+        'external',
+        // 'gateway_delegated',
+        // 'gateway_builtin',
+      ],
+    )
+  }
+
+  status() {
+    return this.faker.helpers.arrayElement(
+      [
+        'not_available',
+        'partially_degraded',
+        'offline',
+        'online',
+      ],
+    )
+  }
+
+  protocol() {
+    return this.faker.helpers.arrayElement(
+      [
+        'http',
+        'grpc',
+        'tcp',
+        'kafka',
+      ],
+    )
+  }
+}
+export default class FakeKuma extends Faker {
+  kuma = new KumaModule(this)
+}
```

Comment:
We don't use this just yet, but we will, and we'll probably expand on this.

This file gives faker a utility of kuma specific things/words